packages/medusa/src/models/payment-session.ts

```diff
@@ -1,11 +1,17 @@
-import { BeforeInsert, Column, Entity, Index, JoinColumn, ManyToOne, Unique, } from "typeorm"
+import {
+  BeforeInsert,
+  Column,
+  Entity,
+  Index,
+  JoinColumn,
+  ManyToOne,
+  Unique,
+} from "typeorm"
 
 import { BaseEntity } from "../interfaces"
-import { Cart } from "./cart"
-import { DbAwareColumn, resolveDbType } from "../utils/db-aware-column"
 import { generateEntityId } from "../utils"
-import { FeatureFlagDecorators } from "../utils/feature-flag-decorators"
-import OrderEditingFeatureFlag from "../loaders/feature-flags/order-editing"
+import { DbAwareColumn, resolveDbType } from "../utils/db-aware-column"
+import { Cart } from "./cart"
 
 export enum PaymentSessionStatus {
   AUTHORIZED = "authorized",
```

Comment:
todo: @olivermrbl did you look at the one line 25?

Reply:
Nice catch!

Reply:
btw, this is a good reminder to always add the feature flag name on these comments.

Reply:
@carlos-r-l-rodrigues found it because it does a bit. Regexp todo.*order

Reply:
Yes, indeed! We should think of a general way of naming todos, because I was mistaking this for a normal comment

Reply:
`// [MEDUSA_FF_ORDER_EDIT] TODO: This could be a way to do it`

Reply:
Maybe something to add to the documentation in notion todo: [flag_key] ?

Reply:
Not sure all ide works with something before todo. My ide list me all todo when it starts with todo and the color is green instead of grey

Reply:
Upvote for `// TODO [flag_key]`. 
Jetbrain's IDE has many workflows for comments starting with `todo` (case-insensitive).